src/test/ui/generic-associated-types/issue-88595.rs

```diff
@@ -0,0 +1,22 @@
+#![feature(generic_associated_types)]
+#![feature(type_alias_impl_trait)]
+
+fn main() {}
+
+trait A<'a> {
+    type B<'b>: Clone
+    where
+        'a: 'b, Self: 'a, Self: 'b;
```

Comment:
Oh uh...right. can you add a FIXME(generic_associated_types) here to remove one of these bounds? It's currently unresolved for the outlives lint and I don't want to miss this after that is resolved.